BoldyrevEV/analyzer.js

```diff
@@ -0,0 +1,32 @@
+var fs = require('fs');
```

Comment:
Не стоит смешивать var и let/const
```suggestion
const fs = require('fs');
```